src/main/java/org/spongepowered/api/entity/living/ArmorStand.java

```diff
@@ -132,18 +132,18 @@
     void setSmall(boolean small);
 
     /**
-     * Returns whether this armor stand is rendered invisible.
+     * Returns whether this armor stand is affected by gravity or not.
      *
-     * @return Whether this armor stand is invisible
+     * @return Whether this is affected by gravity or not
      */
-    boolean isInvisible();
+    boolean hasNoGravity();
 
     /**
-     * Sets whether this armor stand is invisible or not.
+     * Returns whether this armor stand is affected by gravity or not.
```

Comment:
The previous Javadocs were correct, this sets something, but doesn't return something.